### PR TITLE
supermagic mpi sanity code

### PR DESCRIPTION
**Pull request by jordiblasco**

### CODE REVIEW

**Comment by boegel** [comment]
Jenkins: test this please

File: easybuild/easyconfigs/s/supermagic/supermagic-20130104-goolf-1.5.14.eb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# This is an easyconfig file for EasyBuild, see https://github.com/hpcugent/easybuild
+# Authors::   Jordi Blasco <jordi.blasco@nesi.org.nz>
+# License::   New BSD
+
```
Comment (boegel) [inline easybuild/easyconfigs/s/supermagic/supermagic-20130104-goolf-1.5.14.eb]: missing `easyblock = 'ConfigureMake'`

File: easybuild/easyconfigs/s/supermagic/supermagic-20130104-goolf-1.5.14.eb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+description = """Very simple MPI sanity code. Nothing more, nothing less."""
+
+toolchain = {'name': 'goolf', 'version': '1.5.14'}
+toolchainopts = {'usempi': True}
+
+git_commit_id = '943a7f4703'
+sources = ['%s.tar.gz' % git_commit_id]
+source_urls = ['https://github.com/hpc/supermagic/archive']
+
+preconfigopts = "./autogen && "
```
Comment (boegel) [inline easybuild/easyconfigs/s/supermagic/supermagic-20130104-goolf-1.5.14.eb]: this implies adding `Autogen` and/or `Automake` (and `libtool`?) as *build* dependencies

@wpoely86: how do you determine which one(s) are needed?

Comment (wpoely86) [inline easybuild/easyconfigs/s/supermagic/supermagic-20130104-goolf-1.5.14.eb]: Load the autotools bundle as a builddep:
builddependencies = [('Autotools', 'XXX')]

**Comment by boegel** [comment]
@jordiblasco: ping on fixing the remarks?

**Comment by boegel** [comment]
@jordiblasco No interest in getting this merged?

**Pull request closed without merging**